yb-voyager/cmd/importData.go

```diff
@@ -131,7 +131,8 @@ func getYBServers() []*tgtdb.Target {
 			}
 
 			clone.Uri = getCloneConnectionUri(clone)
-			log.Infof("using yb server for import data: %+v", clone)
+			redactedClone := tgtdb.GetRedactedTarget(clone)
+			log.Infof("using yb server for import data: %+v", redactedClone)
```

Comment:
You can combine these two statements:

    log.Infof("...", tgtdb.GetRedactedTarget(clone))